exercises/concept/wizards-and-warriors/WizardsAndWarriorsTests.cs

```diff
@@ -27,6 +27,15 @@ public void Warrior_is_not_vulnerable()
         Assert.False(warrior.Vulnerable());
     }
 
+    [Fact]
+    [Task(3)]
+    public void Wizard_is_not_vulnerable_after_preparing_spell()
```

Comment:
This is the exact same test as the one starting on line 49, and I don't think it concerns Task 3, since up to that point, it doesn't matter the vulnerability of the wizard after preparing a spell.
It might be useful to add another method called IsSpellPrepared() that you could use to test if the state of the spell is actually being saved.